### DISCUSSION TITLE
[Question]:  0.14 - 'Mark these items as reviewed' button - move/duplicate on top of page?

### DISCUSSION THREAD

**Discussion by JoFranke** [blakeblackshear/frigate#13082] [Ask A Question] [answered]
### What is your question:

In the review section, the 'Mark these items as reviewed' button is at the end of the page. 

Sometimes, I have several hundreds of alerts/detections to be reviewed, and I would like to ack them all, but I have to scroll all the way down to click on the button, which is very unconvinient, or I dont know how I can jump to the buttom.

Is there anoter way, or can this button moved/duplicated at the top of page so I can directly ack them all? Thanks.

**Comment by hawkeye217** [comment, marked as answer]
Cmd/Ctrl-A is a helpful keyboard shortcut on desktops to select all review items. You can then just press the “Mark as reviewed” button in the top bar when they’re all selected. For more keyboard shortcuts, check out the FAQ, Usage question 12: https://github.com/blakeblackshear/frigate/discussions/12724

**Reply by JoFranke** [reply]
I'm sorry rtfm - thanks so much! Works like a charm
